packages/widget/src/themes/createTheme.ts

```diff
@@ -0,0 +1,415 @@
+import { loadingButtonClasses } from '@mui/lab';
```

Comment:
I really like having the theme unified and present like this in the one function :-)